Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/StatisticsCardItem.kt

```diff
@@ -2,16 +2,24 @@ package de.rki.coronawarnapp.statistics.ui.homecards.cards
 
 import de.rki.coronawarnapp.statistics.AddStatsItem
 import de.rki.coronawarnapp.statistics.GenericStatsItem
+import de.rki.coronawarnapp.statistics.LocalIncidenceStats
 import de.rki.coronawarnapp.statistics.StatsItem
 import de.rki.coronawarnapp.util.lists.HasStableId
 
 data class StatisticsCardItem(
     val stats: GenericStatsItem,
-    val onClickListener: (GenericStatsItem) -> Unit
+    val onClickListener: (GenericStatsItem) -> Unit,
+    val onRemoveListener: (LocalIncidenceStats) -> Unit,
 ) : HasStableId {
 
     override val stableId: Long = when (stats) {
         is AddStatsItem -> 0L
-        is StatsItem -> stats.cardType.id.toLong()
+        is StatsItem -> {
```

Comment:
I'd prefer to just split the CardItem cleanly here and have one per type, what do you think?
Neither of the listeners is valid for the `AddStatsItem` card and `onRemoveListener` is only valid for the local incidence stats.